### PR TITLE
Update our debugging instructions.

### PR DESCRIPTION
**Pull request by seanlip**
This PR attempts to streamline and simplify our debugging instructions, in particular, clarifying the instructions for what to do in the event of an error in the CI checks.

### CODE REVIEW

File: Debugging-Docs.md [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
+Debugging docs also make it easy for you to bring someone else up to speed on your bug. For example, once, a member of the dev workflow team was debugging an end-to-end test flake which was blocking nearly all PRs from being merged, but they had to go to sleep before they could figure it out. They sent the debugging doc to another contributor who read through the document, quickly spotted a problem, and worked with another team member to create a PR up to fix the issue before the first member woke up the next morning! The debugging doc made it easy for team members to hand the bug off and get it fixed fast.
 
 Finally, these documents can serve as records of debugging strategies and bug causes that we can reference later on. For example, we might search these debugging docs for an error message we are encountering to see if it has been fixed before.
-
-## Examples
```
Comment (seanlip) [inline Debugging-Docs.md:18]: Note: I moved these to the debugging doc template.

File: If-CI-checks-fail-on-your-PR.md [hunk 2]
```diff
@@ -1,96 +1,95 @@
...
+* [What to do if the failure is due to your PR](#what-to-do-if-the-failure-is-due-to-your-pr)
+* [What to do if the failure is due to an existing issue](#what-to-do-if-the-failure-is-due-to-an-existing-issue)
+* [What to do if CI checks only fail in the merge queue](#what-to-do-if-ci-checks-only-fail-in-the-merge-queue)
 
 ## Introduction
 
 If your PR build fails, do not despair! Scroll down to the bottom of the PR thread until you see the results of the continuous integration (CI) tests that ran:
 
-![Screenshot of PR CI results](images/prCiResults.png)
+  ![GHCI Status](images/ghciSample.png)
```
Comment (U8NWXD) [inline If-CI-checks-fail-on-your-PR.md]: "GHCI Status" might be confusing as alt text. Maybe "Screenshot of CI results on GitHub"?

File: If-CI-checks-fail-on-your-PR.md [hunk 3]
```diff
@@ -1,96 +1,95 @@
...
-    "of %i bytes" % (read, size), result)
-urllib.ContentTooShortError: retrieval incomplete: got only 7372553 out of 18638507 bytes
+Here are some tips for how to determine whether the failing CI check is due to your code or a problem in Oppia's develop branch:
 
-Exited with code exit status 1
-```
+* Always start by **looking at the failure logs**. Click on the "Details" link next to each of the failing tests to inspect their logs.
 
-If you see a different error while installing third-party libraries, please file an issue and mention @automated-qa-reviewers to notify the Automated QA team.
+  ![GHCI Logs](images/githubActionsLogs.png)
```
Comment (U8NWXD) [inline If-CI-checks-fail-on-your-PR.md]: ditto for more informative alt text

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: If-CI-checks-fail-on-your-PR.md (hunk 2 above)
Comment (seanlip) [reply If-CI-checks-fail-on-your-PR.md]: Done.

File: If-CI-checks-fail-on-your-PR.md (hunk 3 above)
Comment (seanlip) [reply If-CI-checks-fail-on-your-PR.md]: Done.

**Comment by seanlip** [comment]
Thanks @U8NWXD! Fixed, PTAL.

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**